frontend/src/component/common/Table/cells/TimeAgoCell/TimeAgoCell.tsx

```diff
@@ -10,19 +10,23 @@ interface ITimeAgoCellProps {
     live?: boolean;
     emptyText?: string;
     title?: (date: string) => string;
+    timestamp?: boolean;
```

Comment:
What about receiving the format function as with the title?
```suggestion
    dateFormat?: (value?: string | number | Date, locale?: string) => string;
```
Because timestamp as a boolean limits you to have 2 formats, if you want to add a third one things gets complicated. If the function is not provided you just use the default formatting function:
```typescript
const fn = dateFormat || formatDateYMD;
const date = fn(value, locationSettings.locale);
```